README.md

```diff
@@ -1,29 +1,45 @@
 # erlfmt
 
 erlfmt is an opinionated Erlang code formatter. By automating the process of
-formatting the code, it frees your team up to focus on what the code does
+formatting the code, it frees your team up to focus on what the code does,
 instead of how it looks like.
 
-erlfmt enforces a consistent style by parsing your code and re-printing it with
-it's own rules. When re-printing the code, erlfmt runs an optimisation
-algorithm to find the best layout possible. It will traverse all the possible
-layouts to find the shortest one (in terms of lines of code) where each line
-fits within the selected maximum line-length.
+## Before
 
-For example, the following snippet:
+Remember reading code before erlfmt and having arguments with co workers :(
 
-```erl formatted hello
-hello(mike, joe, robert)
+```erl
+what_is(erlang) ->
```

Comment:
Should `Erlang` be a variable here?

Reply:
Yes it should.  Good catch.  I will correct that.

Reply:
Fixed